demo/auth/authService.js

```diff
@@ -0,0 +1,39 @@
+var fh = require('fh-js-sdk');
+
+function AuthService() {
+}
+
+AuthService.prototype.getProfile = function($http, $window) {
+  var req = {
+    method: 'GET',
+    url: fh.getCloudURL()+ '/profile'
+  };
+  return $http(req, {withCredentials: true}).then(function(res) {
+    return res.data;
+  }, function(err) {
+    if (err.status === 401) {
+      $window.location = fh.getCloudURL() + '/login';
+    }
+    if (err.status === 403) {
+      console.log('Forbidden')
```

Comment:
Any forbidden call will be visible in console anyway, but none of the applications users sees console. Why not use logger? IMHO Cordova "event dialog" will be perfect for this.